### PR TITLE
add os gem update prompt to show username

### PR DESCRIPTION
**Pull request by jacobherrington**
Tested on Mac, should work on Windows/Linux as well.

Open to improvements.

Refs #15

### CODE REVIEW

File: lib/packages/prompt.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 require 'colorized_string'
+require 'os'
 
 def display_prompt
+  print ColorizedString[(OS.windows? ? ENV["USERNAME"] : ENV["USER"]) + "@"].colorize(:yellow)
```
Comment (jacobherrington) [inline lib/packages/prompt.rb]: @faraazahmad thoughts?

Comment (faraazahmad) [inline lib/packages/prompt.rb]: Give username display another function. that'd be cleaner

File: lib/packages/prompt.rb [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+require 'os'
 
 def display_prompt
+  print ColorizedString[get_user].colorize(:yellow)
   print ColorizedString["SRB "].colorize(:yellow)
   print ColorizedString["#{Dir.pwd}"].colorize(:blue)
   print "> "
 end
+
+def get_user
```
Comment (jacobherrington) [inline lib/packages/prompt.rb]: @faraazahmad opinions? Happy to change

Comment (faraazahmad) [inline lib/packages/prompt.rb]: ```display_user``` would be fine for now

**Pull request merged**